src/main/java/com/google/step/snippet/external/GeeksForGeeksClient.java

```diff
@@ -52,6 +52,6 @@ public Card search(String geeksLink) {
     String title = titles.first().text();
     String description = descriptions.first().text();
     String code = snippets.first().getElementsByClass(CODE_CLASS).text();
-    return new Card(title, code, geeksLink, description);
+    return new Card(title, code, geeksLink, description, "GeeksForGeeks");
```

Comment:
Thoughts on making "GeekForGeeks" and the other source names constants?

I know in #36, @anbenson mentioned that "Login" and "Logout" don't need to be constants since it's only used in one place in the file. However, this case feels _slightly_ different since from looking at `new Card(title, code, geeksLink, description, "GeeksForGeeks")` it isn't 100% clear what "GeeksForGeeks" means to Card. I think `new Card(title, code, geeksLink, description, SOURCE_NAME)` or something might make it easier to understand?

Whereas in #36, `request.setAttribute(AUTH_LABEL, "Login")` makes it quite clear what "Login" is being used for.